### PR TITLE
Removed old policy topics from beta and added redirects

### PR DESCRIPTION
**Pull request by dkershaw10**
Fix for #8798

### CODE REVIEW

File: api-reference/beta/api/application-delete-tokenissuancepolicies.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: "Remove tokenIssuancePolicy"
```
Comment (Lauragra) [inline api-reference/beta/api/application-delete-tokenissuancepolicies.md:2]: These three new API topics need to be surfaced in the beta TOC.

Comment (dkershaw10) [reply api-reference/beta/api/application-delete-tokenissuancepolicies.md:2]: Yeah - you are correct.  Will fix.  I only discovered they were missing when removing the old topics.

**Comment by dkershaw10** [comment]
@Lauragra Any reason this wasn't merged?

**Comment by Lauragra** [comment]
> @Lauragra Any reason this wasn't merged?

Yes, the reason is that builds were taking 30-45 mins per PR again yesterday and I didn't have enough time to get each PR merged in sequentially before I had to go to bed. :| If this keeps happening we might need to look into a better way to manage the growing volume of PRs. I will try to get the remaining PRs merged in throughout the day today.

**Comment by dkershaw10** [comment]
Thanks Laura!

**Pull request merged**